ci/azure-pipelines.yml

```
@@ -114,6 +114,26 @@ stages:
             displayName: Build on Ubuntu
             condition: contains(variables['build_container'], 'ubuntu')
 
+          - bash: |
+              set -eEx
+              ./autogen.sh
+              export CFLAGS='-Werror -Wall'
+              export KFLAGS='-Werror'
```

Can you please remove KFLAGS for both?

---

also since it rebuilds existing tree I guess we'd need to git clean. currently CI did not rebuild it.

---

It is fixed now. I somehow missed it. Thank you for spotting that.

---

also since it rebuilds existing tree I guess we'd need to `make distclean` before make. currently CI did not rebuild it.

---

Fixed. Didn't realize that each one was done right after another in place (new to azure pipelines).

---

CI error is due to directory that has been already removed, I guess we can simply `make distclean || :`.

---

I changed the line to be `make -i distclean || true` so that `make` ignores errors and that the exit code is always using `true` instead of `:` (makes it more clear)

---

Well, that cleaned out the `Makefile` too, so had to add `./configure` again after it.

---

Now all CI checks pass

---

i see ;) eventually maybe it's simpler to use like `test -f Makefile && make -i distclean || true` before autgen.sh

---

I am not sure which is better. It probably makes little difference